<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>studentview</class>
 <widget class="QDialog" name="studentview">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(162, 0, 0);</string>
       </property>
       <property name="text">
        <string>ID :</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignLeft">
      <widget class="QLabel" name="ID_label">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(162, 0, 0);</string>
       </property>
       <property name="text">
        <string>Name :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Name_label">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(162, 0, 0);</string>
       </property>
       <property name="text">
        <string>Year :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Year_label">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(162, 0, 0);</string>
       </property>
       <property name="text">
        <string>Group :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Group_label">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="NC_lable">
       <property name="styleSheet">
        <string notr="true">font: 700 16pt &quot;Perpetua&quot;;
color: rgb(162, 0, 0);</string>
       </property>
       <property name="text">
        <string>number of courses : </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="TA_label">
       <property name="styleSheet">
        <string notr="true">font: 700 16pt &quot;Perpetua&quot;;
color: rgb(162, 0, 0);</string>
       </property>
       <property name="text">
        <string>total absences : </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(175, 198, 192);
font: 600 12pt &quot;Segoe UI Semibold&quot;;</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <column>
      <property name="text">
       <string>course</string>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>197</red>
         <green>27</green>
         <blue>30</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>absence</string>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>197</red>
         <green>27</green>
         <blue>30</blue>
        </color>
       </brush>
      </property>
     </column>
     <column>
      <property name="text">
       <string>average</string>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="foreground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>197</red>
         <green>27</green>
         <blue>30</blue>
        </color>
       </brush>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="styleSheet">
        <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
border-radius: 15px;
margin-left: 4px; border-radius: 15px; background: rgba(49, 45, 52, 253); color: #4A0C46;
color : white;
 border-style: outset;
     border-width: 2px;
     border-radius: 15px;
     border-color: beige;
     font: bold 13px;
     min-width: 4em;
     padding: 5px;</string>
       </property>
       <property name="text">
        <string>edit exam</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 900 12pt &quot;Segoe UI&quot;;
</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(162, 0, 0);</string>
       </property>
       <property name="text">
        <string>Total average:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="average">
       <property name="styleSheet">
        <string notr="true">font: 700 18pt &quot;Perpetua&quot;;
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="styleSheet">
      <string notr="true">font: 900 11pt &quot;Segoe UI Black&quot;;
border-radius: 15px;
margin-left: 10px; border-radius: 15px; background: rgba(49, 45, 52, 253); color: #4A0C46;
color : white;
 border-style: outset;
     border-width: 2px;
     border-radius: 15px;
     border-color: beige;
     font: bold 13px;
     min-width: 4em;
     padding: 5px;</string>
     </property>
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
